lib/Runtime/PlatformAgnostic/Platform/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ set(PL_SOURCE_FILES
   Linux/UnicodeText.ICU.cpp
   Linux/NumbersUtility.cpp
   Linux/Thread.cpp
+  Linux/PerfTrace.cpp
```

Comment:
well, not sure what do you mean by OSX compatible. I just don't know how this output will be useful there.